src/theories/Category.jl

```diff
@@ -44,6 +44,7 @@ symbol ``⋅`` (`\\cdot`) for composition in diagrammatic order.
 end
 
 # Convenience constructors
+compose(f) = f
```

Comment:
Is this needed? Preferable to use `compose(::AbstractVector)` for programmatic use.